test/builtin-vars-gep.ll

```diff
@@ -13,29 +23,61 @@ target triple = "spir64"
 ; Function Attrs: alwaysinline convergent nounwind mustprogress
 define spir_func void @foo() {
 entry:
+; ORIG:   %GroupID = alloca [3 x i64], align 8
+; ORIG:   %0 = addrspacecast ptr addrspace(1) @__spirv_BuiltInWorkgroupSize to ptr addrspace(4)
+; ORIG:   %1 = getelementptr <3 x i64>, ptr addrspace(4) %0, i64 0, i64 0
+; ORIG: ; CHECK-LLVM %[[GLocalSize0:[0-9]+]] = call spir_func i64 @_Z14get_local_sizej(i32 0) #1
+; ORIG:   %2 = addrspacecast ptr addrspace(1) @__spirv_BuiltInWorkgroupSize to ptr addrspace(4)
+; ORIG:   %3 = getelementptr <3 x i64>, ptr addrspace(4) %2, i64 0, i64 2
+; ORIG:   %4 = load i64, ptr addrspace(4) %1, align 32
+; ORIG:   %5 = load i64, ptr addrspace(4) %3, align 8
+; ORIG: ; CHECK-LLVM %[[GLocalSize2:[0-9]+]] = call spir_func i64 @_Z14get_local_sizej(i32 2) #1
+; ORIG: ; CHECK-LLVM  mul i64 %[[GLocalSize0]], %[[GLocalSize2]]
+; ORIG:   %mul = mul i64 %4, %5
+; ORIG:   ret void
```

Comment:
that's confusing, shouldn't we leave just CHECK-LLVM ?

Reply:
This is the test as it was in original commit: with opaque pointers.

There are two ways to backport this change: 
1. Either find and port the other changes related to opaque pointers support on LLVM -> SPIRV translation stage, then we can use this original test with -opaque-pointers options added to RUN: commands. 
2. Backport this commit only, but with the test changed not to use opaque pointers (which is currently happening in this pull request).

Reply:
I suggest to leave only used in the current patch on the LLVM 14 (15, 16, 9) CHECK-LLVMs removing ORIG lines.

Reply:
Sorry, do you suggest removing 
1. All ORIG lines
2.  Only those containing ORIG: ; CHECK-LLVM
 ? 
 Please notice that  CHECK-LLVM are not triggered as they have no ":" at the end of the token

Reply:
I suggest to remove all ORIG lines as this patch is not about opaque pointers and ORIG lines contains them, it's a minor comment - this very test on this patch is read only anyway. Feel free to not apply it. Yet would be nice to update the description for the PR itself saying what is needed from the original patch.